modules/timepoint_list/php/timepoint_list.class.inc

```diff
@@ -149,14 +149,6 @@ class Timepoint_List extends \NDB_Menu
                 $this->tpl_data['timePoints'][$x]['currentDate']
                     =   $timePoint->getCurrentDate();
 
-                // get Center Name
```

Comment:
See Timepoint class change on line 130 for justification of deletion

https://github.com/aces/Loris/pull/4919/files#diff-3331857654de63c280eeedba8a7fca9cR130

@driusan sorry i couldnt help myself deleting 10 lines to replace it by 10 characters